mxcube3/ui/components/PopInput/DefaultInput.jsx

```diff
@@ -8,14 +8,19 @@ export default class DefaultInput extends React.Component {
 
   constructor(props) {
     super(props);
+    this.state = {value: this.props.value};
     this.save = this.save.bind(this);
     this.cancel = this.cancel.bind(this);
     this.submit = this.submit.bind(this);
+    this.onChange = this.onChange.bind(this);
+  }
+  
+  onChange(event) {
+    this.setState({value: event.target.value});
```

Comment:
I have  already made a fix for this there is no PR yet. My fix does not involve setState which I think we should avoid.
Is there a reason for the setState other than the obvious one to retrieve the value ?